localStorage.js

```diff
@@ -0,0 +1,18 @@
+const form = document.querySelector('.formSection');
+
+const {name, email, msg } = form.elements;
+if (!localStorage.getItem('formObjectData')) {
+  const formData = { fnameData: '', emailData: '', messageData: '' };
+  localStorage.setItem('formObjectData', JSON.stringify(formData));
+}
```

Comment:
- [ ] Kindly notice that what the previous code reviewer meant was that the data that the user has entered should be visible in the contact form when the user reloads the page, so I suggest that you add this piece of code to load the data when the page loads.

```javascript
if (!localStorage.getItem('formObjectData')) {
  const formData = { fnameData: '', emailData: '', messageData: '' };
  localStorage.setItem('formObjectData', JSON.stringify(formData));
} else {
  const formData = JSON.parse(localStorage.getItem('formObjectData'));
  name.value = formData['fnameData'];
  email.value = formData['emailData'];
  msg.value = formData['messageData'];
}
```

Reply:
Thanks for the clarification and for help.
you can now check it . 👋 
@ShahierNashaat

Reply:
Should I request a review ?

Reply:
Yes

Reply:
I request a review. thanks a lot 🌷